pygmt/tests/test_grdcontour.py

```diff
@@ -88,14 +90,14 @@ def test_grdcontour_interval_file_full_opts(grid):
     fig = Figure()
 
     comargs = {
-        "region": [-161.5, -154, 18.5, 23],
+        "region": [-53, -49, -20, -17],
         "interval": TEST_CONTOUR_FILE,
         "grid": grid,
         "resample": "100",
         "projection": "M10c",
         "cut": 10,
     }
-    fig.grdcontour(**comargs, limit=(-25000, -1), pen=["a1p,blue", "c0.5p,blue"])
+    fig.grdcontour(**comargs, limit=(0, 400), pen=["a1p,blue", "c0.5p,blue"])
```

Comment:
Do you think there is a bug here? From how I interpret the docs, `limit=(0, 400)` means that contours above 400 should not be drawn but the image looks like there are contours up to at least 800.

Reply:
No, it's from user error on my part; the next command just had a lower limit of 0, so everything above it was plotted. I rewritten the test.